src/portal.tsx

```diff
@@ -34,12 +34,12 @@ export const Portal: React.FC<Props> = ({children, className}) => {
 
     React.useEffect(() => {
         if (container && className) {
-            container.classList.add(className);
+            container.classList.add(...className.split(' '));
         }
 
         return () => {
             if (container && className) {
-                container.classList.remove(className);
+                container.classList.remove(...className.split(' '));
```

Comment:
check this out, now we need this, VE could pass more than one class here (LoadingBar needs this)

Reply:
good catch